csm/core/services/s3/accounts.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, s3plugin, provisioner):
         self._s3_root_client = IamRootClient()
 
     @Log.trace_method(Log.DEBUG, exclude_args=['password'])
-    async def create_account(self, account_name: str, account_email: str, password: str):
+    async def create_account(self, account_name: str, account_email: str, password: str) -> dict:
```

Comment:
Almost there!

Using `dict` as a type is now supported in Python 3.9. Unfortunately it is not in Python 3.6 :/ We have to use `typing.Dict`.

Reply:
It's not actually my idea. I could see `dict` as a returning type almost everywhere in this file. Do you think we should fix all of them?

Reply:
Yes, probably. But not with this patch, let's just make this one right.

Reply:
Ok, done.